### PR TITLE
Add brother Printer Setting Tool 1.1.4a & 1.1.4

### PR DESCRIPTION
**Pull request by puttin**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-drivers/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/brother-printer-setting-tool.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask 'brother-printer-setting-tool' do
+  if MacOS.version >= '10.15'
```
Comment (vitorgalvao) [inline Casks/brother-printer-setting-tool.rb]: Always do `if`’s with the release name and with the `else` for the current version. So here make it `if MacOS.version <= :mojave` and the `else` covers Catalina.

File: Casks/brother-printer-setting-tool.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    # https://download.brother.com/welcome/dlfp100791/pstm114ax13all.dmg
```
Comment (vitorgalvao) [inline Casks/brother-printer-setting-tool.rb]: Remove the comments.

File: Casks/brother-printer-setting-tool.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    version '1.1.4a,13:dlfp100791'
```
Comment (vitorgalvao) [inline Casks/brother-printer-setting-tool.rb]: Are you sure the `dlfp100791` will keep changing?

Comment (puttin) [reply Casks/brother-printer-setting-tool.rb]: @victorpopkov this part is a copycat of https://github.com/Homebrew/homebrew-cask-drivers/blob/master/Casks/brother-p-touch-update-software.rb and the version below 10.15 actually has a different `dlfpxxxx`. 

so yes I think.

Comment (vitorgalvao) [reply Casks/brother-printer-setting-tool.rb]: > So we may need to set different url with different macOS version.

Which is perfectly supported and done in plenty of casks.

> That's kind of annoying to me.

How’s that annoying to you? Once it’s setup, thank to interpolation it takes exactly the same trouble to update as if it were a single URL.

> I think keeping the pattern is good.

Not relying on `colon` is better.

Comment (puttin) [reply Casks/brother-printer-setting-tool.rb]: done

Comment (vitorgalvao) [reply Casks/brother-printer-setting-tool.rb]: Thank you. If it turns out those change, we can modify the cask.

**Pull request merged**